core/src/main/java/io/aiven/klaw/service/SchemaRegistrySyncControllerService.java

```diff
@@ -161,9 +168,33 @@ private List<SchemaSubjectInfoResponse> getSchemasFromDb(
         schemaInfo.setTeamId(topic.getTeamId());
 
         schemaInfoList.add(schemaInfo);
+        topicSchemaVersionsInDb.remove(topic.getTopicname());
       }
     }
+    return schemaInfoList;
+  }
 
+  private List<SchemaSubjectInfoResponse> annotateAnomalousSchemasNotInDB(
+      String schemaEnvId,
+      List<SchemaSubjectInfoResponse> schemaInfoList,
+      Map<String, Set<String>> topicSchemaVersionsInDb,
+      Map<String, SchemaInfoOfTopic> schemaTopicAndVersions) {
+    // Annotate anomalous Schemas
+    Set<Map.Entry<String, Set<String>>> schemaNames = topicSchemaVersionsInDb.entrySet();
```

Comment:
```suggestion
    Set<Map.Entry<String, Set<String>>> schemaVersionsDbSet = topicSchemaVersionsInDb.entrySet();
```
just better naming

Reply:
Slightly different, -> schemaTopicNameToVersionsDB I dont think we need to call it a set as that is already its type.